src/components/smart_objects/src/smart_object.cc

```diff
@@ -723,16 +723,28 @@ void SmartObject::duplicate(const SmartObject& OtherObject) {
 void SmartObject::cleanup_data() {
```

Comment:
@LuxoftAKutsan Yeah, this modification is not relative with logic request controller, because this modification is too small changes to send another PR, so I sent this PR here, please reference [comment](https://github.com/smartdevicelink/sdl_core/pull/660#discussion_r69402044). If really need send another PR for this, let me know. Thanks!